codefori/vscode-ibmi
# SEP support

### Changes

* [x] Add new attribute for batch debugging (#1823)
* [x] Add new SEP port config for connections (#1893)
* [x] Pass in port and SEP port when starting debug service (#1894)
* [x] Change Java versions based on installed Debug Service (#1841)
* [x] Support starting batch and SEP debug (#1824)
   * [x] Code written to execute the SEP debug command
   * [x] UI/Commands updated/created to launch batch/SEP debug from active editor and object browser
* [x] https://github.com/codefori/vscode-ibmi/issues/1932
* [x] Ensure debug extension is activated before trying to launch SEP
* [x] Merged in #1895 (this PR depends on it)

### How to test this PR

1. [x] Ensure using Debug Service version 1 still works (✅ by @worksofliam)
2. [x] Show error if SEP is not supported (✅ by @worksofliam)
3. [x] Can launch batch from active editor (✅ by @worksofliam)
3. [x] Can launch batch from Object Browser (✅ by @mkwan01)
4. [x] Can launch SEP from active editor (✅ by @mkwan01)
5. [x] Can launch SEP from Object Browser (✅ by @mkwan01)

### Checklist
